src/backend/kvm/config.rs

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+
+pub struct Config(());
```

Comment:
Could this just be `pub struct Config;`? Or is the `()` necessary because of the `Sized` constraint on `trait Config`?

Reply:
...ah, I've just been informed that this is an idiomatic way of having a zero-sized struct that's `pub`, but can't actually be instantiated outside the crate/module. Kinda threw me off for a minute, and it's not very googleable - could you add a comment here mentioning the reason for the `()`?